src/assets/text/scrollyText.js

```diff
@@ -6,71 +6,67 @@ export default {
+          text: 'Drought is increasing worldwide, threatening our water supplies and affecting our daily lives. What starts with reduced rain can progress to impacts on soil moisture and streamflow.'
         },
         {
           // id is used to source image files with the same naming
           // and used to assign classes to scrolling text
           id: 'aa02',
-          text: 'text aa02: Drought often begins as meterological drought, which means that there is less rain or snow melt than expected.'
+          text: 'When there is less rainfall or snowmelt than typical, this can cause meteorological drought.'
         },
         {
           // id is used to source image files with the same naming
           // and used to assign classes to scrolling text
           id: 'aa03',
-          text: 'text aa03: Meterological drought can lead to agricultural drought, which is where soil moisture becomes depleted and our crops and livestock begin to suffer and farmers rely on using irrigation.'
+          text: 'When soil moisture becomes depleted, agricultural drought sets in. To keep crops and livestock healthy, farmers use more water for irrigation.'
         },
         {
           // id is used to source image files with the same naming
           // and used to assign classes to scrolling text
           id: 'aa04',
-          text: 'text aa04: Prolonged agricultural drought can begin to cause hydrological drought, which affects deeper soil layers and eventually reduces streamflow. Streamflow drought is one type of hydrological drought.'
+          text: 'As these conditions persist, less water moves into and through streams. When streamflow levels are unusually low, this is defined as streamflow drought. But what does "unusually low" really mean?'
         },
         {
           // id is used to source image files with the same naming
           // and used to assign classes to scrolling text
           id: 'a',
-          text: 'text a: Although we have been hearing a lot about drought in the western US lately, drought can happen anywhere. For example, here\'s one severe streamflow drought that occurred at a site in Ohio in late summer of 1963.'
+          text: 'Consider this Ohio streamgage site in the late summer of 1963. On August 15th there was a severe streamflow drought.'
         },
         {
           id: 'b',
-          text: 'text b: Streamflow droughts like this occur whenever streamflow gets below a certain threshold. The threshold can be defined based on typical streamflow rates over the year or by day. Let\’s explore how changing the threshold affects when we detect streamflow droughts at this site.'
+          text: 'We can call streamflow unusually low whenever streamflow drops below a certain level or threshold. Here, drought happened when flow dropped below 45 cubic feet per second (cfs). Why 45 cfs? There are two methods to choose that threshold, depending on the questions we are trying to answer.'
```

Comment:
```suggestion
          text: 'We can say that streamflow is 'unusually low' whenever streamflow drops below a certain level or threshold. Here, that threshold was defined as 45 cubic feet per second (cfs). Why 45 cfs? There are two methods to choose that threshold, depending on the questions we are trying to answer.'
```

Reply:
^SC

Reply:
SC: Looks like the term method is now only used here and at the end, and the individual 'methods' aren't called out as such. Maybe the word method should be used throughout or dropped? Also, unless we say what questions each methods answers, I'm not sure that bit makes sense to include.